test/cypress/support/commands.js

```diff
@@ -444,8 +443,11 @@ Cypress.Commands.add('giRedirectToGroupChat', () => {
 })
 
 Cypress.Commands.add('giWaitUntilMessagesLoaded', () => {
+  cy.get('.c-initializing').should('not.exist')
   cy.getByDT('conversationWrapper').within(() => {
-    cy.get('.c-initializing').should('not.exist')
+    cy.get('.infinite-status-prompt:first-child')
+      .invoke('attr', 'style')
+      .should('include', 'display: none')
```

Comment:
`.c-initializing` is always existing while loading the first page of the channel. It works just like a skeleton and it could be replaced with the awesome skeleton component later.
So when user opens or switches channels, it is always there in the chat so we should wait until it would be disappeared.
While fixing the Cypress error in this PR, I noticed that `.c-initializing` doesn't exist **inside** the `conversationWrapper` element. So I take it out.

And `.infinite-status-prompt:first-child` is the infinite loading spinner. In many cases of our test scenario, we don't make many messages which can't fill even one page. So infinite loading tries to load more and also we should wait until it is fully done.

Reply:
I tried 5 times in row after update this code, Cypress didn't break.